dashboard/views/addEditClassroom.ejs

```diff
@@ -14,7 +14,7 @@
                             <div class="col-md-12">
                                 <div class="form-group form-black label-floating">
                                     <label class="control-label" data-l10n-id="name"></label>
-                                    <input class="form-control" name="name" value="<% if(typeof classroom == 'object'){ %><%= classroom.name %><% } %>" type="text">
+                                    <input class="form-control" name="name" value="<% if(typeof classroom == 'object'){ %><%= classroom.name %><% } %>" type="text" required>
```

Comment:
I'm not in favor of using "required". It's not consistent with other forms plus it don't use the same localization than the server. You could just let the server do the work.